### DISCUSSION TITLE
problem with Touch ID in MBP 13.2 with a touchbar from 2016

### DISCUSSION THREAD

**Discussion by djpetere** [community/community#69632] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Maybe someone is interested in the problem with Touch ID in MBP 13.2 with a touchbar from 2016. Many users report the problem that the touch ID settings are not visible at all. you need to change the settings in SMBIOS Spoof but this does not solve the problem. Settings cannot be modified. If anyone is interested in the problem, I will be happy to provide more details.

**Comment by Piero24** [comment]
U can try with this [guide](https://www.easeus.com/computer-instruction/macbook-pro-touch-bar-not-working.html). This is usually correlated to some security issue. Maybe you have changed some settings in recovery mode or with sudo privilege. A similar problem appears when you try to use apple pay on a mac with touch ID but you have disabilited the security settings.
